documentation/contribute/add_a_method.qmd

```diff
@@ -3,14 +3,14 @@ title: Add a method
 order: 40
 ---
 
-This page describes how to add a method to an already existing task. Make sure you have followed the [Requirements](requirements.qmd) and [first setup](first_setup.qmd) pages.
+This page describes how to add a method to an already existing task. Make sure you have followed the [Requirements](requirements.qmd) and [getting started](getting_started.qmd) pages.
 
 There are 2 type of components you can add:
 
 * `control method`:  The control methods are methods that form a baseline (random/ground truth) to compare the methods from the same task against.
 * `method`: scripts to be benchmarked on how to perform the task.
 
-You can find the components in their respective directories namely the `control_methods` and `methods` directory. The setup of both components is similar with small differences in the config yaml. 
+You can find the components in their respective task directories (`src/task_name/`) namely the `control_methods` and `methods` directory. The setup of both components is similar with small differences in the config yaml. 
 The differences will be pointed out below. whenever `methods` is mentioned it should be replaced with `control_methods` when adding a control method.
```

Comment:
Move this to a separate section at the bottom, so people aren't immediately confronted with the baseline methods?